css-three-column-layout/index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Three Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <div class="row">
+      <div class="column-full">
+        <header>
+          <h2>Designed for life science. Optimized for SaMD and Digital Health.</h2>
+        </header>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-one-third">
+        <div class="image-container">
+          <img src="images/merge.png" alt="Merge icon">
+        </div>
+      </div>
+      <div class="column-one-third">
+        <div class="image-container">
+          <img src="images/cloud-computing.png" alt="Cloud computing icon">
+        </div>
+      </div>
+      <div class="column-one-third">
+        <div class="image-container">
+          <img src="images/discussion.png" alt="Discussion icon">
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-one-third">
+        <h3>Streamlined QMS</h3>
+        <p>A QMS that includes modules for all stages of the product life cycle from Design Control to CAPA</p>
+      </div>
+      <div class="column-one-third">
+        <h3>Integrates with existing tools</h3>
+        <p>Enzyme QMS integrates with your existing tools, such as JIRA, GitHub and Jenkins with Enzyme</p>
+      </div>
+      <div class="column-one-third">
+        <h3>In-depth Onboarding</h3>
+        <p>We train your entire team and provide you with support when and as it's needed</p>
+      </div>
+    </div>
```

Comment:
Try reorganizing this so that each column contains the image as well as the heading and paragraph.